packages/web/tests/components/VideoTableRow.test.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { render, screen } from '../testUtils/testTools';
+import { VideoTableRow } from '../../src/components/VideoTableRow';
+import { Video } from '../../../api/src/entities/Video';
```

Comment:
Same thing here, instead of importing from `api` import the interface you'll make [here](https://github.com/AmericanAirlines/Project-X/pull/32/files#r724243361)